src/components/profile/UserProfile.js

```diff
@@ -0,0 +1,70 @@
+import { ArrowBack } from '@styled-icons/boxicons-regular'
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useParams } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+import { getUserById } from '../../utils/api'
+import { useAppHistory } from '../../utils/useAppHistory'
+import { PageScrollWrapper, PageTemplate } from '../about/PageTemplate'
+import BackButton from '../ui/BackButton'
+import { LoadingOverlay } from '../ui/LoadingIndicator'
+
+const StyledNavBack = styled.div`
+  svg {
+    height: 20px;
+    margin-right: 5px;
+  }
+`
+const UserProfile = () => {
+  const { id } = useParams()
+  const history = useAppHistory()
+  const { t } = useTranslation()
+  const [userData, setUserData] = useState({})
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    async function fetchUserData() {
+      setIsLoading(true)
+      const data = await getUserById(id)
+      setUserData(data)
+
+      // Log the fetched data
+      console.log('Fetched User Data:', data)
+
+      setIsLoading(false)
+    }
+
+    fetchUserData()
+  }, [id])
+
+  if (isLoading) {
+    return <LoadingOverlay />
+  }
+
+  const { created_at, name, bio } = userData
+
+  return (
+    <PageScrollWrapper>
+      <PageTemplate>
+        <StyledNavBack>
+          <BackButton
+            onClick={(event) => {
+              event.stopPropagation()
+              history.goBack()
+            }}
+          >
+            <ArrowBack />
+            {t('back')}
+          </BackButton>
+        </StyledNavBack>
+
+        <h1>Name: {name}</h1>
```

Comment:
This is a good start since it shows the data! 

Can we iterate on the styling a bit, and try to use fewer words, so there's less need to translate them?

My original quick attempt was a ` <h2>, user icon for the name, a <p> for a bio, and an italic:`

![Screenshot from 2024-11-11 10-35-39](https://github.com/user-attachments/assets/f4de791b-55ec-42a2-ac15-986a0f02d008)



![Screenshot from 2024-11-11 10-34-32](https://github.com/user-attachments/assets/a0660eb8-a76c-44c7-91d9-5f4d9fcb8269)

but I'm not great at styling pages. 

Maybe borrow the styling from the import page?
It looks like
```
<h3> #301: Dumpster Diving locations (Melbs)</h3>
```
with the rest in `<p>` elements.